requirements.txt

```diff
@@ -0,0 +1,2 @@
+PyJWT==2.0.1
+requests==2.25.1
```

Comment:
Add argparse requirement

Reply:
Argparse is part of the standard library, hence it's not necessary to add argparse as a requirement (that's always why it's not added automagically after running `pip freeze`).